cmd/libs/go2idl/conversion-gen/generators/conversion.go

```diff
@@ -280,8 +280,7 @@ func Packages(context *generator.Context, arguments *args.GeneratorArgs) generat
 			}
 		}
 		if !pkgNeedsGeneration {
-			glog.V(5).Infof("  no viable conversions, not generating for this package")
-			continue
+			glog.Errorf("Package %v has 'k8s:conversion-gen' tags but no viable conversions", i)
```

Comment:
The fact that this wasn't outputting a file was confusing your generated files make system.  I wasn't able to get to a "no op" make without some sort of fix here.